docs/download.md

```diff
@@ -81,29 +94,27 @@ CUDA 11.8 through CUDA 12.0.
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
 * Verify the signature for Scala 2.12 jar:
-    `gpg --verify rapids-4-spark_2.12-23.12.2.jar.asc rapids-4-spark_2.12-23.12.2.jar`
+    `gpg --verify rapids-4-spark_2.12-24.02.0.jar.asc rapids-4-spark_2.12-24.02.0.jar`
 * Verify the signature for Scala 2.13 jar:
-    `gpg --verify rapids-4-spark_2.13-23.12.2.jar.asc rapids-4-spark_2.13-23.12.2.jar`
+    `gpg --verify rapids-4-spark_2.13-24.02.0.jar.asc rapids-4-spark_2.13-24.02.0.jar`
 
 The output of signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Introduced support for chunked reading of ORC files.
-* Enhanced support for additional time zones and added stack function support.
-* Enhanced performance for join and aggregation operations.
-* Kernel optimizations have been implemented to improve Parquet read performance.
-* RAPIDS Accelerator also built and tested with Scala 2.13.
-* Last version to support Pascal-based Nvidia GPUs; discontinued in the next release.
-* Introduced support for parquet Legacy rebase mode (spark.sql.parquet.datetimeRebaseModeInRead=LEGACY and spark.sql.parquet.int96RebaseModeInRead=LEGACY)
-* Introduced support for Percentile function.
-* Delta lake 2.3 support.
+* Discontinued support for Nvidia GPUs based on Pascal architecture.
+* Set get_json_object functionality to disabled by default.
+* Implemented string comparison in AST expressions.
+* Expanded timezone support to include options beyond UTC.
+* Enhanced security by adding checksum for cached files in Filecache.
+* Introduced support for Databricks 13.3 ML LTS.
+* Added support for parse_url functionality.
 * Qualification and Profiling tool:
```

Comment:
The qualification and profiling tools are released separately and on an independent schedule.  Does it make sense to list things here vs. point to the tools download page and let it describe what the recent releases are and what they have?

Reply:
How about referring to https://github.com/NVIDIA/spark-rapids-tools/releases @mattahrens @amahussein  ?